common/get-host-info.sub.js

```diff
@@ -18,10 +21,12 @@ function get_host_info() {
     ORIGINAL_HOST: ORIGINAL_HOST,
     REMOTE_HOST: REMOTE_HOST,
 
+    ORIGIN: PROTOCOL + "//" + ORIGINAL_HOST + PORT_ELIDED,
```

Comment:
WPTs are served on HTTP by default. These don't seem to be needed because there is no ambiguity.

Reply:
When run on the bots, sure. But on w3c-test.org, they can be accessed from both http and https. This makes the test more robust, so I prefer to keep it (unless WPT folks object for some reason)